front_end/html/view_answer.html

```diff
@@ -1,21 +1,33 @@
 {% if problem_basics["exists"] %}
     {% if problem_details["show_answer"] or assignment_details["view_answer_late"] %}
         {% if not last_submission and not assignment_details["due_date"] %}
-            <p>You don't have any submissions for this exercise yet. You won't be able to view the instructor's code until you've submitted the correct answer.</p>
+            <p>You don't have any submissions for this exercise yet. You won't be able to view the instructor's solution until you've submitted the correct answer.</p>
```

Comment:
This is nitpicky, but will you please change all instances (throughout all of the code) of "don't" to "do not" and "won't" to "will not" and "you've" to "you have." That will be more friendly to people for whom English is not their first language, etc.